libraries/AP_Scripting/AP_Scripting.h

```diff
@@ -35,6 +35,10 @@ class AP_Scripting
 
     bool enabled(void) const { return _enable != 0; };
 
+    void offer_message(const mavlink_message_t &msg, const mavlink_channel_t chan);
```

Comment:
maybe a comment to clarify what "offer_" means?  For new people coming in it might be good to clarify